content/millennium/r4/individuals/practitioner.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Practitioner | R4 API
+---
+
+# Practitioner
+
+* TOC
+{:toc}
+
+## Overview
+
+The Practitioner Resource provides information about a person formally involved in the care of a patient on behalf of a healthcare facility. Practitioners include and are not limited to physicians, nurses, pharmacists, therapists, technologists, and social workers.
```

Comment:
Should this be 'Practitioners include **but** are not limited to?

```suggestion
The Practitioner Resource provides information about a person formally involved in the care of a patient on behalf of a healthcare facility. Practitioners include but are not limited to physicians, nurses, pharmacists, therapists, technologists, and social workers.
```

Reply:
changed: https://github.com/cerner/fhir.cerner.com/pull/252/commits/114b72f1ff982efcb5315405c67849dae19fcfc8#diff-00a306a2aa1334ff6f799b4200c75030R12